keyboards/handwired/meck_tkl/blackpill_f401/mcuconf.h

```diff
@@ -0,0 +1,24 @@
+/* Copyright 2020 QMK
```

Comment:
I think this can be deleted, as it doesn't really do anything other than include the base mcuconf for this board. Same with the halconf.